src/diffcalc_API/models/response.py

```diff
@@ -0,0 +1,29 @@
+from typing import Dict, List
+
+from pydantic import BaseModel
+
+from diffcalc_API.models.ub import HklModel
+
+
+class InfoResponse(BaseModel):
+    message: str
```

Comment:
There's an argument to be made that each endpoint should have a unique response model. Just because they're all the same now, doesn't mean they will always be, and if one of them changes it will affect things downstream. On the other hand it's a lot more code. Food for thought, but happy to approve as-is.

Reply:
Yeah, that's a good point. For now, I've left it as a bare minimum for the openAPI schema to just recognize the return types of each endpoint.